Cargo.toml

```diff
@@ -131,6 +131,7 @@ binary_macros = "=1.0.0"
 fancy-duration = "=0.9.2"
 stringreader = "=0.1.1"
 tempfile = "=3.10.0"
+glob = "=0.3.1"
```

Comment:
I don't think you should add glob just for this, beaware of library/feature creep.

Reply:
It's added in the `[dev-dependencies]` section just for testing.

I usually don't mind dependencies that are there just for tests, do you still think it's too much?

Reply:
Also found out that it's already included in our `Cargo.lock` file.